src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 4 failures in `PowerShell-CI-windows`
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1:170 char:101
+ ... ject 1 && Write-Object ([System.Collections.Generic.List]::new((2,3))
+                                                                          ~
Missing closing ')' in expression.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1:170 char:101
+ ... ject 1 && Write-Object ([System.Collections.Generic.List]::new((2,3))
+                                                                          ~
Missing closing ')' in expression.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1:170 char:101
+ ... ject 1 && Write-Object ([System.Collections.Generic.List]::new((2,3))
+                                                                          ~
Missing closing ')' in expression.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1:170 char:101
+ ... ject 1 && Write-Object ([System.Collections.Generic.List]::new((2,3))
+                                                                          ~
Missing closing ')' in expression.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```